Sample Applications/Win11ThemeGallery/README.md

```diff
@@ -1 +1,6 @@
-## WPF Win11 Theme Gallery App
+## WPF Fluent Windows Gallery Application
+The application is a control catalog, containing various controls, their variations and appearance in FluentWindows style. The application is still a work in progress, with occasional irregularities in sample codes of some controls.
+
+Developers need to do the following changes in order to use this application:
+1. Update `sdk`'s `version` to latest 9 preview, or the one available in local system, in `global.json` present in the repository root.
+2. Build a local WPF repository and update the `WpfRepoRoot` with it's path in `Win11ThemeGallery.csproj` file.
```

Comment:
Developers will need more guidance for this. For the first step, just mention that you need to install latest .NET 9 preview SDK.

Reply:
Will update it